src/pages/Dashboard/MintRepayVai/MintVai/index.tsx

```diff
@@ -141,7 +145,7 @@ const MintVai: React.FC = () => {
     if (!account) {
       // This error should never happen, since the form inside the UI component
       // is disabled if there's no logged in account
-      throw new Error('An internal error occurred: account undefined. Please try again later.');
+      throw new Error(t('mintRepayVai.mintVai.errorMessage'));
```

Comment:
What do you think about making this error reusable? We use this exact phrasing inside the v2 components for all internal components, so we could have something like that:
```typescript
throw new Error(t('errors.internal', { errorMessage: t('mintRepayVai.mintVai.undefinedAccountErrorMessage') }));
```

The error would look like that in the translation file:
```json
"errors": {
  "internal": "An internal error occurred: {{errorMessage}}. Please try again later."
}
```

Reply:
yeah. nice suggestion